packages/test/test-end-to-end-tests/src/test/attachRegisterLocalApiTests.spec.ts

```diff
@@ -7,22 +7,24 @@ import { strict as assert } from "assert";
 import { IRequest, IFluidCodeDetails } from "@fluidframework/core-interfaces";
 import { AttachState } from "@fluidframework/container-definitions";
 import { Loader } from "@fluidframework/container-loader";
-import { IUrlResolver } from "@fluidframework/driver-definitions";
 import {
     LocalCodeLoader,
     ITestFluidObject,
     TestFluidObjectFactory,
     TestFluidObject,
     createDocumentId,
     LoaderContainerTracker,
+    ITestObjectProvider,
 } from "@fluidframework/test-utils";
 import { SharedObject } from "@fluidframework/shared-object-base";
 import { IContainerRuntimeBase } from "@fluidframework/runtime-definitions";
 import { SharedMap } from "@fluidframework/map";
 import { requestFluidObject } from "@fluidframework/runtime-utils";
 import { ChildLogger } from "@fluidframework/telemetry-utils";
+import { describeNoCompat } from "@fluidframework/test-version-utils";
 
-describe(`Attach/Bind Api Tests For Attached Container`, () => {
+// REVIEW: enable compat testing?
```

Comment:
Does this need proactive follow-up? (I'm not a fan of sprinkling these comments in the code... they often don't get looked at and end up stale, not to mention ugly/distracting when reading the code)

Reply:
I just want to document that there no particular reason that we don't enable compat test.  But there is no urgency to do them, and no need to have a proactive follow-up, so I don't want to create an issue.

Any alternatives you have in mind?

Reply:
Nah, that makes sense. To me, labelling the comment "REVIEW" seems to indicate some importance, so I wanted to check in case we did want to be sure to follow up.